site/src/app.js

```diff
@@ -43,7 +43,10 @@ app.use((state, emitter) => {
     state.secure = false
     state.ws = new util.WS(state.params.host)
 
-    state.ws.on('open', () => state.secure = state.ws.secure)
+    state.ws.on('open', () => {
+      state.secure = state.ws.secure
+      emitter.emit('emotes.fetch')
```

Comment:
Note that this should probably be auto-fetched periodically (or when a new `emotes update` event is sent over the WS) on clients. At the moment you need to reload in order to see newly-added emoji.